app/main.py

```diff
@@ -1 +1,44 @@
-#  write your code here
+class Animal:
+    def __init__(self
+                 , name: str
+                 , appetite: int
+                 , is_hungry: bool = True) -> None:
```

Comment:
Place each argument including `self` on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability. Do not place commas on the first position, place them on the last position in line instead